packages/realm/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "realm",
   "version": "12.2.1",
-  "description": "Realm is a mobile database: an alternative to SQLite and key-value stores",
+  "description": "Realm by MongoDB is an offline-first mobile database: an alternative to SQLite and key-value stores",
```

Comment:
local first? @ianpward what terms are we using these days?

Reply:
Spoke to Ian, he said that "offline-first" is what we've traditionally gone with.

Reply:
Maybe we can add "local first" to the keywords?